packages/core/src/Stores/contract-store.js

```diff
@@ -381,24 +381,25 @@ export default class ContractStore extends BaseStore {
                 ? { take_profit: getLimitOrder(this).take_profit }
                 : getLimitOrder(this);
 
-        WS.contractUpdate(this.contract_id, limit_order).then(response => {
-            if (response.error) {
-                this.root_store.common.setServicesError({
-                    type: response.msg_type,
-                    ...response.error,
-                });
-                return;
-            }
+        Object.keys(limit_order).length !== 0 &&
```

Comment:
Do we need a sanity of V1?

Reply:
No, this is just a precautionary measure